website/integrations: grafana: migrate to entitlements (#21676)
goauthentik/authentik: website/integrations/monitoring/grafana/index.mdx

    - Set the Logout URI to `https://grafana.company/logout`.
    - Set the Logout Method to `Front-channel`.
    - Select any available signing key.
    - Under **Advanced protocol settings** > **Selected Scopes**, add `authentik default OAuth Mapping: OpenID 'entitlements'`.
- **Configure Bindings** _(optional)_: you can create a [binding](/docs/add-secure-apps/bindings-overview/) (policy, group, or user) to manage the display and access to applications on a user's **My applications** page.

3. Click **Submit** to save the new application and provider.

### Create application entitlements for Grafana roles

> Use [application entitlements](/docs/add-secure-apps/applications/manage_apps/#application-entitlements) to represent the Grafana roles that this application should assign.

Reviewer: ```suggestion
Use [application entitlements](/docs/add-secure-apps/applications/manage_apps/#application-entitlements) to represent the Grafana roles that can be assigned to users.
```
